keps/sig-api-machinery/1693-warnings/README.md

```diff
@@ -109,38 +109,39 @@ In client-go:
    3. log warnings with klog.Warn
 3. Add a process-wide warning handler (defaulting to the klog.Warn handler)
 4. Add a per-client warning handler (defaulting to the process-wide warning handler)
-
+ 
 In kubectl, configure the per-process handler to:
 
 1. dedupe warnings (only print a given warning once per invocation)
 2. log to stderr with a `Warning:` prefix
 3. color the `Warning:` prefix if stderr is a terminal and `$TERM != "dumb"` and `$NO_COLOR` is unset
```

Comment:
@sttts are these "normal"?  I have no idea.

Reply:
@BenTheElder has a lot of experience here in making pretty terminal output for terminals that can support it

Reply:
These are "normal" 😅 
https://no-color.org/
https://en.wikipedia.org/wiki/Computer_terminal#Dumb_terminals